app/helpers/application_helper.rb

```diff
@@ -55,4 +55,12 @@ def t_fallback(key, options = {})
       false
     end
   end
+
+  def make_bold_substring(string, substring)
+    if (string.is_a? String) && (substring.is_a? String) && string.include?(substring)
+      string.gsub(substring, "<strong>#{substring}</strong>").html_safe
+    else
+      string
+    end
```

Comment:
I'd suggest looking at the[ Rails `highlight` method](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-highlight).

Reply:
Nice 👍